packages/core/build/build.tokens.ts

```diff
@@ -97,3 +194,22 @@ function convertCSSValue(token: Token, base = 20, fallback = true) {
 function camelCaseToKebab(val: string) {
   return val.replace(/([A-Z]|\d+)/g, '-$1').toLocaleLowerCase();
 }
+
+function isHSL(value: any) {
+  return Array.isArray(value) && value.length === 3;
```

Comment:
This is probably fine for now but I'm wondering what it would do if it were passed an array of rgb values? HSL is `[0-359][0-100][0-100]`. RGB is `[0-255][0-255][0-255]`

I don't see how this would catch that.

Perhaps leave a note for a possible future cleanup? Or would it be unnecessary here because we are controlling all values?

Reply:
Yeah since we control the values it wasnt something I thought needed to be checked. However I think in the future we can work towards making this script more generic to the point of it being its own cds package with the token formats.